octodns_constellix/__init__.py

```diff
@@ -48,7 +48,17 @@ def __init__(self, log, api_key, secret_key, ratelimit_delay=0.0):
         self.api_key = api_key
         self.secret_key = secret_key
         self.ratelimit_delay = ratelimit_delay
-        self._sess = Session()
+        self._sess = CachedSession(
+            backend='memory',
+            allowable_methods=['GET', 'HEAD'],
+            allowable_codes=[200],
+            ignored_parameters=[
+                'x-cns-security-token',
```

Comment:
@ross @viranch I will unify and simplify the request handling for `ConstellixClient` and `SonarClient` later on in a different PR.